CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+| wontfix             | Marks an item that will not be fixed                                                               |
+
+
+### Milestone glossary
+
+A common set of milestones, described in this section, are used across repositories managed by the Azure Functions team. A given repository may use additional milestones. Be sure to check that repository's contribution guide or ask a maintainer if you have any questions.
+
+| Milestone                     | Description                                                                                                                                                                       |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Active Questions / Discussion | Refers to items where someone is actively investigating or answering a question. Items in this milestone should always have an owner.                                             |
+| Backlog                       | Refers to items that may need additional design or are expected to be delivered outside of the next 6 months                                                                      |
+| Sprint _X_                    | Refers to a specific work sprint during which this should be resolved                                                                                                             |
+| Triaged                       | Refers to items for which design is mostly understood and for which delivery should be within a 6 month period. Items in this milestone may just be waiting on sprint assignment. |
+| Unknown                       | Refers to any item for which no guarantees can be made. This is often a place for discussions and design.                                                                         |
+
+## Submitting a pull request
+
+Azure Functions welcomes pull requests. However, we ask that if you wish to address an issue tracked in one of our repositories that is not flagged with the `help-wanted` label, please check in with the maintainers to make sure work is not already in progress.
+
+Your pull request will now go through extensive checks by the project maintainers. Please be patient; the team operates a number of repositories and is often busy with their own contributions and service maintenance. We will try to get to your pull request as soon as we can. Please note that you may be asked to rebase or target a different branch as part of the review process, especially if other changes have been merged which would create a conflict.
+
+The general flow for making a change to an Azure Functions repository is:
+1. 🍴 Fork the repo (add the fork via `git remote add me <clone url here>`
+2. 🌳 Create a branch for your change (generally use dev) (`git checkout -b my-change`)
+3. 🛠 Make your change
+4. ✔️ Test your changes
```

Comment:
Where do we teach people how to test their changes? This is the biggest blocker IMO.

Reply:
That's going to be per-repo. This is just the first PR to set the stage, and then we'll need to touch all repos we maintain.

Reply:
I'm not sure what you mean by set the stage. If there will be per-repo instructions, I advise you call that out here, as in something like:

> Test your changes (instructions for testing will be in the repo-specific CONTRIBUTING guide).

This makes it clear to the customer where to go for the information they need.